wiki/BBCode/de.md

```diff
@@ -116,11 +111,11 @@ Text
 
 Das Tag `[box]` wird verwendet, um Texte und Bilder in einem anklickbaren Hyperlink zu verstecken. Beim Klicken wird der Inhalt in der Box ähnlich wie in einem Dropdown-Menü aufgedeckt.
 
-Der benutzerdefinierte Hyperlink wird durch das Argument `NAME` bestimmt. Die Spezifizierung dieses Arguments wird eine Überschrift in der Box anhand des Arguments erstellen und wird die Größe der Box entsprechend anpassen. Wenn die Spezifizierung nicht erfolgt, dann wird das Tag `[box]` standardmäßig eine Box ohne Überschrift erstellen (auf die nicht geklickt werden kann). Das Argument benutzt keine Anführungszeichen (`"`) und stellt Leerzeichen dar.
+Der benutzerdefinierte Hyperlink wird durch das Argument `NAME` bestimmt. Die Spezifizierung dieses Arguments wird eine Überschrift in der Box anhand des Arguments erstellen und wird die Größe der Box entsprechend anpassen. Wenn `NAME` nicht definiert wird, dann erstellt das Tag `[box]` eine Box ohne Überschrift. Das Argument benutzt keine Anführungszeichen (`"`) und stellt Leerzeichen dar.
 
 Das Tag wird meistens benutzt, um große Fluten an Text und Bildern zu verstecken, die den Forumsbeitrag immens verlängern. Am meisten sichtbar in den FAQ oder in [Skin](/wiki/Skinning)-Veröffentlichungsbeiträgen.
 
-*Anmerkung: die Schaltfläche der BBCode-Box in der Symbolleiste wird "spoiler box" genannt, aber erstellt keinen `[spoilerbox]` Tag.*
+*Anmerkung: Die Schaltfläche der BBCode-Box in der Symbolleiste wird "spoiler box" genannt, aber sie erstellt keinen `[spoilerbox]` Tag.*
```

Comment:
Looking at [crowdin](https://crowdin.com/translate/osu-web/12/en-de#136), the button text is translated as `Spoiler`. Should I change this to

```suggestion
*Anmerkung: Die Schaltfläche der BBCode-Box in der Symbolleiste wird "Spoiler" genannt, aber sie erstellt keinen `[spoiler]` Tag.*
```

so that it corresponds to the UI text and better differentiates between box, spoiler, and spoilerbox?

Alternative:

```suggestion
*Anmerkung: Die Schaltfläche der BBCode-Box in der Symbolleiste wird "Spoiler" genannt, aber sie erstellt keinen `[spoiler]` oder `[spoilerbox]` Tag.*
```

Reply:
Probably makes sense to go by the UI text, would go with your 2nd suggestion then.

Reply:
Yes, you're right, it's `kein` due to `Tag` being neutral.

I'm prefering the second option as well to be honest.